sdk/daml-script/runner/src/main/scala/com/daml/lf/engine/script/v1/ScriptF.scala

```diff
@@ -231,6 +231,12 @@ object ScriptF {
       } yield SEAppAtomic(SEValue(continue), Array(SEValue(SList(res))))
 
   }
+
+  def makePair(v1: SValue, v2: SValue): SValue = {
+    import com.daml.script.converter.Converter.record
+    record(StablePackagesV2.Tuple2, ("_1", v1), ("_2", v2))
+  }
+
```

Comment:
Is this not defined anywhere else for us to use? Perhaps in converter?

edit:I see you moved it from lower down in the file, so I'm guessing the answer to my question is no

Reply:
Right, I couldn't find it anywhere else but I'm happy to move it there. I'm not sure which `Converter` you're talking about though, is it:
  - daml-script/runner/src/main/scala/com/daml/lf/engine/script/Converter.scala
  - daml-script/runner/src/main/scala/com/daml/lf/engine/script/v1/Converter.scala
  - or daml-script/converter/src/main/scala/com/daml/script/converter/Converter.scala

?

Reply:
Theres converter/Converter, as well as a versioned Converted for v1 and v2. Really just wondering if it exists in any of these, seems a common builder.

Reply:
Moved to `daml-script/runner/src/main/scala/com/daml/lf/engine/script/Converter.scala` as discussed offline.